Apps/Patient/test/unit/Controllers/PatientDataControllerTests.cs

```diff
@@ -0,0 +1,41 @@
+// -------------------------------------------------------------------------
+//  Copyright © 2019 Province of British Columbia
+//
+//  Licensed under the Apache License, Version 2.0 (the "License");
+//  you may not use this file except in compliance with the License.
+//  You may obtain a copy of the License at
+//
+//  http://www.apache.org/licenses/LICENSE-2.0
+//
+//  Unless required by applicable law or agreed to in writing, software
+//  distributed under the License is distributed on an "AS IS" BASIS,
+//  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+//  See the License for the specific language governing permissions and
+//  limitations under the License.
+// -------------------------------------------------------------------------
+
+using System;
+using System.Text.Json;
+using HealthGateway.Patient.Services;
+using Xunit;
+
+namespace HealthGateway.PatientTests.Controllers
+{
+    public class PatientDataControllerTests
+    {
+        [Fact]
+        public void CanSerizlizePatientData()
+        {
+            var data = new PatientData[]
+            {
+                new OrganDonorRegistrationData("Registered", "Message", Guid.NewGuid().ToString())
+            };
+
+            var response = new PatientDataResponse(data);
+
+            var serialized = JsonSerializer.Serialize(response);
+
+            Assert.NotNull(serialized);
```

Comment:
In `PatientDataServiceTests` you use `serialized.ShouldNotBeNullOrEmpty` is there a risk of false positives due to empty string?

Reply:
It may be because I wrote it before adding Shoudly. I can fix that.